website/pages/en/substreams/[[...slug]].mdx

```diff
@@ -22,20 +22,32 @@ export async function getStaticProps({ params: { slug = ['README'] } }) {
   if (response.status === 404 && paths !== 'README') {
     response = await fetch(`${baseURL}${paths}/README.md`)
   }
-  const data = await response.text()
+  const data = (await response.text())
+    // replace {% embed ... %} with <iframe />
+    .replaceAll(
+      /{%\s+embed\s+url="(.*?)"\s+%}/g,
+      (...m) =>
+        `<iframe src="${m[1].replace(
+          // we need enhance YouTube links, otherwise they will be not loaded in iframe
+          'youtube.com/watch?v=',
+          'youtube.com/embed/'
+        )}" style={{aspectRatio: 16/9, width: '100%'}}/>`
+    )
+    // remove gitbook {% ... %} elements
+    .replaceAll(/{%.*?%}/g, '')
+    // close img tags only if he doesn't point to .gitbook
+    .replaceAll(/<img(.*?)>/g, (...m) => (m[1].includes('src=".gitbook') ? '' : `<img${m[1]}/>`))
+    // fixes http://localhost:3000/en/substreams/reference-and-specs/authentication/
+    .replaceAll('<pre class="language-bash" data-overflow="wrap"><code class="lang-bash">', '```bash\n')
```

Comment:
we need remove `<pre />` and `<code />` from here https://github.com/streamingfast/substreams/blob/develop/docs/reference-and-specs/authentication.md?plain=1#L40-L45